### PR TITLE
Documentation clarification

### PR DESCRIPTION
**Pull request by lesyk**

### CODE REVIEW

**Comment by arogal** [comment]
Hi Lesyk. Please fill out https://docs.google.com/a/uber.com/forms/d/1pAwS_-dA1KhPlfxzYLBqK6rsSWwRwH95OCCZrcsY5rk/viewform for us to accept this PR. Thanks!

File: README.md [hunk 1]
```diff
@@ -88,6 +88,8 @@ Add the following code snippet, replacing the placeholders with your app’s inf
 <string>[redirect URL]</string>
 ```
 
+Make sure UberCallbackURI mathes one, which is set on developer dashboard (use localhost for testing).
```
Comment (Lupie) [inline README.md]: Mathes?

**Comment by lesyk** [comment]
Filled.

**Comment by jbrophy17** [comment]
Hi @lesyk thanks for contributing! I think this could even clearer, maybe something like "Make sure the value for UberCallbackURI exactly matches one of the Redirect URLs you have set on your developer dashboard. (You can use localhost for testing)" What do you think?

**Comment by lesyk** [comment]
Done, + small change.

File: README.md [hunk 2]
```diff
@@ -87,8 +87,7 @@ Add the following code snippet, replacing the placeholders with your app’s inf
 <key>UberCallbackURI</key>
 <string>[redirect URL]</string>
 ```
-
-Make sure UberCallbackURI matches one, which is set on developer dashboard (use localhost for testing).
+Make sure the value for UberCallbackURI exactly matches one for the Redirect URLs you have set on your developer dashboard. (You can use `localhost` for testing.)
```
Comment (jbrophy17) [inline README.md]: Small typo still, we want "of" instead of "for" in "...exactly matches one of the Redirect URLs..."

**Comment by jbrophy17** [comment]
Just one other small comment, then good to go

**Comment by jbrophy17** [comment]
Looks good!
One last thing, I've just pushed a development branch (0.5.x-dev) do you think you could actually make this PR against that branch instead? Makes it easier to see whats going into the next release vs what's already out.
Thanks!

**Comment by jbrophy17** [comment]
@lesky any update here? You're going to need to open a new PR against the 0.5.x-dev branch

**Pull request closed without merging**